package.json

```diff
@@ -18,11 +18,14 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-typescript": "^7.3.3",
     "@types/jest": "^24.0.18",
+    "@types/react": "^16.8.0",
```

Comment:
it's not necessary here.

Reply:
react and redux too

Reply:
your need run `npm i` from `packages/packageName/`. Adding them here is not the solution.